Integration/XverifyIntegration.php

```diff
@@ -181,7 +181,7 @@ public function doEnhancement(Lead $lead)
                                     $status  = $this->getResponseStatus($response, $fieldKey);
                                     if (!is_null($status)) {
```

Comment:
There it is ... i was checking to see which was needed since they both appeared to have the identical effect...